mixer/adapter/kubernetesenv/kubernetesenv.go

```diff
@@ -299,3 +351,52 @@ func newKubernetesClient(kubeconfigPath string, env adapter.Env) (k8s.Interface,
 	}
 	return k8s.NewForConfig(config)
 }
+
+func (b *builder) createCacheController(k8sInterface k8s.Interface, clusterID string) error {
```

Comment:
You could consider adding a unit test to check these create and delete functions perform properly.

Reply:
the createCacheController and deleteCacheController are called in the unit test. I verify that they are called by testing the length of the b.controllers data structure.